faery/csv.py

```diff
@@ -15,27 +15,34 @@ class CsvEventStreamIterator(StreamIterator):
 
     CSV_PATTERN = re.compile(r"(\d*),(\d*),(\d*),(\d)")
 
+    buffer: List[Tuple[int, int, int, bool]]
```

Comment:
`buffer: List[Events]` generates an Exception when importing the library, as Events is not a standard type but a Numpy datatype. This solves the problem, like it's done in the `StdEventInputIterator` class.

Reply:
Great catch! Interesting about the exception, shouldn't that be reproduced in the GitHub actions test? I noticed all the checks were passing. Is there something we could/should add as a test to catch errors like this?